src/questions.js

```diff
@@ -0,0 +1,84 @@
+const questions = [
+  // Title
+  {
+    name: "title",
+    message: "What's the title of your project?",
+    type: "input",
+  },
+  // Description
+  {
+    name: "description",
+    message: "Type a description for your readme",
+    type: "input",
+  },
+  //   Needs Installation?
+  {
+    name: "installation",
+    message: "Does your project require an installation?",
+    type: "confirm",
+    default: "true",
+  },
+  //   Installation description
+  {
+    name: "installationText",
+    message: "What's the installation process?",
+    type: "input",
+    when: (answers) => answers.installation,
+  },
+  //   Usage
+  {
+    name: "usage",
+    message: "How does a user use your application?",
+    type: "input",
+    when: (answers) => answers.installation,
+  },
+  //   Testing
+  {
+    name: "test",
+    message: "How does a user test your application?",
+    type: "input",
+    when: (answers) => answers.installation,
+  },
```

Comment:
As mentioned in the feedback, I'd add the useage and test questions even if the use says no to the installation. Consider making these questions necessary.